blog/MeetthehoodiesNickHehr.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: 'Meet the Hoodies: Nick Hehr'
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+#Hello all! I recently had the great honor of interviewing Nick Hehr. Let's see what he had to say:
+
+#1. What’s your favorite animal?
+
+While I’m a massive fan of dogs, yet another reason why I love the Hoodie mascot, I’ve always been fascinated by owls. It probably started after watching The Many Adventures of Winnie the Pooh as a kid. Owls, as I’ve seen them, are usually portrayed as wise, knowledgeable creatures and good friends. Plus, how can you not love Hedwig’s Theme from the Harry Potter movies?
+
+
+#2. What’s your role / what have you contributed to Hoodie?
+
+I’ve contributed to Hoodie in quite a few ways over the years. Ever since meeting Gregor online and learning a bit about the project, I’ve been a great fan and proponent. I started speaking about it at local meetups and creating a few demo apps as I was learning more about web development. My first direct contribution to Hoodie was through writing app tutorials for the documentation, which helped me dig into how it all worked under the hood (no pun intended). From there, I got involved with the latest Hoodie rewrite after some encouragement from amazing Hoodie core team. I began contributing front-end code and discussing architectural decisions, learning tons about how to maintain an inclusive, open-source community. Lately, I’ve been helping maintain the various Hoodie repos by reviewing issues and welcoming new contributors through Meet the Hoodies events. Best part of my day is merging a PR from a new contributor and welcoming them to this wonderful project.
```

Comment:
Be sure to refer to the glossary of common terms in the editorial wiki when referring to frontend and open source, etc. https://github.com/hoodiehq/editorial/wiki/Glossary-of-common-terms